### DISCUSSION TITLE
Zeek Intel Not Working

### DISCUSSION THREAD

**Discussion by ColeVan** [Security-Onion-Solutions/securityonion#5333] [Unsupported Versions] [answered]
After following new documentation for zeek intel configuration the intel.log is now empty. I've verified no errors in reporter.log as well as verified that intel.dat properly transferred to docker. Anyone else having this issue after upgrading from 2.3.61 to 2.3.70?

**Comment by dougburks** [comment, marked as answer]
Zeek Intel is working for me on 2.3.70:

![Screen Shot 2021-08-31 at 5 57 08 AM](https://user-images.githubusercontent.com/1659467/131482776-507efdec-799d-46f0-8e42-ebe4a6660195.png)

Are you sure that Zeek is seeing traffic that matches your intel?

**Reply by ColeVan** [reply]
Doug,

Zeek is indeed capturing the domain I'm looking for. This is in the last 10 mins. I checked tab spacing and everything looks correct.
![image](https://user-images.githubusercontent.com/70167373/131484907-35385f51-e95d-474c-b606-bad845664539.png)

**Reply by dougburks** [reply]
In your screenshot, it looks like the first line of your intel.dat defines 3 fields but the second line has 4 fields.

**Reply by ColeVan** [reply]
After correcting the file and copying the intel directory to correct location I still see no logs. The domains are indeed seen by zeek. The reporter log seems odd. Any thoughts? Any updates to zeek in 2.3.70 that may have broken the intel framework features?
![image](https://user-images.githubusercontent.com/70167373/131493738-0bb42dfb-2bc9-4f60-8ccf-a4f35f234d10.png)

![image](https://user-images.githubusercontent.com/70167373/131493662-9ff880ac-66c5-4f5c-842a-0b8e6f3bfed7.png)

**Reply by ColeVan** [reply]
Update

I was able to get everything working properly. The intel.dat file is very specific on tab and syntax. It would be great to get a demo from someone on how to properly integrate MISP with IP, Domain, and URL IOC's so that we can leverage zeek intel framework.
![image](https://user-images.githubusercontent.com/70167373/131588297-bd8fc3d8-1949-4347-a794-31fe2c473b2d.png)

**Reply by dougburks** [reply]
I'm glad it's working for you now @ColeVan.

**Reply by weslambert** [reply]
> Update
> 
> I was able to get everything working properly. The intel.dat file is very specific on tab and syntax. It would be great to get a demo from someone on how to properly integrate MISP with IP, Domain, and URL IOC's so that we can leverage zeek intel framework.
> ![image](https://user-images.githubusercontent.com/70167373/131588297-bd8fc3d8-1949-4347-a794-31fe2c473b2d.png)

Hi @ColeVan , 

For MISP integration, you might consider using something like the following to grab Zeek Intel and Suricata NIDS rules:

https://github.com/weslambert/securityonion-misp

**Reply by ColeVan** [reply]
@weslambert Thanks! I'm currently in the process of figuring out how to import c2 IP's, domains, and urls into MISP so that security onion can grab that data and use intel_misp.dat file to notify me in intel dashboard. Your guthub link helped with the initial setup between security onion and MISP instance.
